src/System.Private.CoreLib/src/System/Diagnostics/Eventing/RuntimeEventSource.cs

```diff
@@ -50,6 +51,7 @@ protected override void OnEventCommand(System.Diagnostics.Tracing.EventCommandEv
                     // overhead by at all times even when counters aren't enabled.
                     _counters = new EventCounter[] {
                         // TODO: process info counters
```

Comment:
Can this TODO be removed, or are there more to be added after this one?  Is there an issue number to associate with this?